### PR TITLE
Add validator info for delegators

### PR DESCRIPTION
**Pull request by Pantani**
# Headline
Add validator info for delegators endpoint.

## Problem
/delegations endpoint has missing information about the validator.

[Issue 390](https://github.com/trustwallet/blockatlas/issues/390)

### CODE REVIEW

File: platform/cosmos/api.go [hunk 1]
```diff
@@ -101,19 +104,30 @@ func (p *Platform) GetDelegations(address string) (blockatlas.DelegationsPage, e
...
 	return results, nil
 }
 
-func NormalizeDelegations(delegations []Delegation) []blockatlas.Delegation {
+func NormalizeDelegations(delegations []Delegation, validators blockatlas.ValidatorMap) []blockatlas.Delegation {
 	c := coin.Cosmos()
 	results := make([]blockatlas.Delegation, 0)
 	for _, v := range delegations {
+		validator, ok := validators[v.ValidatorAddress]
+		if !ok {
```
Comment (vikmeup) [inline platform/cosmos/api.go]: Should we add continue here is not found?

Comment (Pantani) [reply platform/cosmos/api.go]: I don't know, this info is crucial to show?

**Comment by Pantani** [comment]
> Maybe we should just merge validators on higher levels than do it for each platform?
> 
> Similar example how we merge github list with platform list

If we merge just the higher level, the delegations for some platforms in production going to break

**Comment by vikmeup** [comment]
Why wound it break? 

We should make it reusable, it does not make sense to merge that on every platform

**Comment by Pantani** [comment]
> Why wound it break?
> 
> We should make it reusable, it does not make sense to merge that on every platform

because of the model changes
https://github.com/trustwallet/blockatlas/pull/395/files#diff-289e7eeff75e55aa7f73ff0d65354063R36

**Comment by vikmeup** [comment]
@Pantani What stopping you from creating temp model before merging? 

I don’t see any issues by improving model structure, otherwise you add complexity for each platform to implement staking protocol api

**Comment by Pantani** [comment]
> @Pantani What stopping you from creating temp model before merging?
> 
> I don’t see any issues by improving model structure, otherwise you add complexity for each platform to implement staking protocol api

I did similar the other PR
https://github.com/trustwallet/blockatlas/pull/383/files

I going to split, no problem

**Pull request closed without merging**